README.md

```diff
@@ -61,15 +61,15 @@ To ensure the proper functioning of the tests on scaffold-stark, your Starknet F
 snforge --version
 ```
 
-If your Starknet Foundry version is not `0.34.0`, you need to install it.
+If your Starknet Foundry version is not `0.35.1`, you need to install it.
 
-- Install Starknet Foundry `0.34.0` via `asdf` ([instructions](https://foundry-rs.github.io/starknet-foundry/getting-started/installation.html#installation-via-asdf)).
+- Install Starknet Foundry `0.35.1` via `asdf` ([instructions](https://foundry-rs.github.io/starknet-foundry/getting-started/installation.html#installation-via-asdf)).
 
 ## Compatible versions
 
 - Starknet-devnet - v0.2.3
 - Scarb - v2.9.2
```

Comment:
this version correct? @Nadai2010

Reply:
Hi @BrunoAmbricca the snfoundry version is 0.34, leave it as it was.

Please check all faults in Contribution and Readme or possible improvements, thank you

Reply:
Hello @Nadai2010. I read the contribution and readme files and didn´t find any faults to improve. Let me know if a miss anything.